_digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx

```diff
@@ -7,102 +7,65 @@ authors: ["Prisha Adya"]
+		This made me even more determined to enter the healthcare field, as I had gained so much interesting knowledge from this experience that had definitely grown my passion for the field.
+		I would seize every learning opportunity I got to learn even more about health sciences!</p>
+	<p>Still not convinced?
+		Here are 5 reasons why joining HOSA would be beneficial to you if you want to pursue a future career in the medical or healthcare field:</p>
+	<h1>1. Meet like-minded individuals</h1>
+		<p>Through HOSA, I have met so many new people and formed so many connections with students who have similar interests as me.
+			If you join HOSA because of your interest in healthcare, science or medicine, chances are that other people joined for these same reasons!
+			At the conference itself, you see hundreds of people who are all there because of their interest in healthcare, which gives you an opportunity to talk to them and connect instantly.</p>
+	<h1>2. Leadership Opportunities</h1>
+		<p>HOSA is a fantastic way for you to improve your leadership skills.
+			By joining HOSA, you develop effective leadership skills through character development and promoting the sense of responsible leadership.
+			In individual or group events especially, being able to compete on your own or leading your team is significant in the development of these skills, so it is a wonderful way to build on your current qualities and skills.
+			For me personally, the improvements in my leadership skills led me to become more confident, which landed me many executive roles in school and out of school.
+			I became a much better speaker and built onto my communication and collaboration skills.
+			I lead many clubs at school now and I do not hesitate to reach out to teachers or potential speakers for our events.</p>
+	<h1>3. Experience</h1>
+		<p>HOSA provided me, along with many other individuals, with extensive hands-on experience, especially in the fields that I plan on pursuing.
+			During my first year in forensics, we trained with a role-play scenario in which we investigated a crime scene and identified specific details such as time of death, how the person was murdered (based on a blood stain sample), and if the person had any type of medication or drugs in their system at the time of murder (through a post - mortem report).
+			During my time in the HOSA bowl category, the amount of information given was amazing.
+			From careers to history in healthcare to specific diseases of every part of the body and how to treat them, the learning opportunity was HUGE!
+			The amount of experience you gain is invaluable.</p>
+	<div align="center">
+			<img src="/blog/january-2021/chalk-outline.png" alt="Chalk outline of a body" width="500"/>
+	</div>
+	<div align="center">
+			<img src="/blog/january-2021/why-you-should-join-hosa/medical-demo.png" alt="HOSA event medical demo" width="500"/>
+	</div>
```

Comment:
Did we want to use `<figure>` around all the images? Some of the images on other blog posts use `<figcaption>`. Up to you (keep as `<img>` or add `<figure>` for styling later)

Reply:
The images from this particular blogpost don't have any captions or other things that would necessitate `<figure>` - I'll probably keep suing the `<img>` tag here.

Reply:
Yep, sounds good. The `<figure>` images are also wrapped with div that centers, don't think there will be any more styling on them.